docs/extensibility.md

```diff
@@ -169,6 +169,8 @@ TAGS="v1"
 
 DH_IMAGE="${HOST}/${USER}/${DH}:${TAGS}"
 
+echo "/target" >> .dockerignore
```

Comment:
should this be "./target"?

Reply:
`.dockerignore` ~== `.gitignore`.

It is treated as the being in the root directory of this repo.

It may be better to use `**/target` though (in both files) so that no `target` directories are ever caught.